docs/UserGuide.md

```diff
@@ -158,6 +197,34 @@ Expected outcome:
 
 ### Sorting internship applications: `sort`
 
+Sorts the list of internship applications on the left panel by the date and time of their tasks by `ORDER`, where 
+`ORDER` is either `a` for ascending or `d` for descending.
+
+Format: `sort ORDER`
+
+* The internships are also sorted with the current date and time taken into consideration.
+* The date of the task that is the earliest and is on or after the current date(if any) in the internship will be used
```

Comment:
```suggestion
* The date of the task that is the earliest and is on or after the current date (if any) in the internship will be used
```
Should there be a whitespace separating the words?

Reply:
Thank you, I have fixed it.